javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlAutoConfiguration.java

```diff
@@ -53,26 +57,29 @@
     private EntityManagerFactory entityManagerFactory;
 
     @Bean
-    public DialectName javersSqlDialectName(){
+    public DialectName javersSqlDialectName() {
         SessionFactoryImplementor sessionFactory =
-                (SessionFactoryImplementor)entityManagerFactory.unwrap(SessionFactory.class);
+                (SessionFactoryImplementor) entityManagerFactory.unwrap(SessionFactory.class);
 
         Dialect hibernateDialect = sessionFactory.getDialect();
         logger.info("detected Hibernate dialect: " + hibernateDialect.getClass().getSimpleName());
 
         return dialectMapper.map(hibernateDialect);
     }
 
-    @Bean(name = "javers")
+    @Bean
     @ConditionalOnMissingBean
-    public Javers javers(ConnectionProvider connectionProvider,
-                         PlatformTransactionManager transactionManager) {
-        JaversSqlRepository sqlRepository = SqlRepositoryBuilder
-                .sqlRepository()
-                .withConnectionProvider(connectionProvider)
-                .withDialect(javersSqlDialectName())
-                .build();
+    public JaversSqlRepository javersSqlRepository(ConnectionProvider connectionProvider) {
```

Comment:
Because we need to expose JaversSqlRepository as a bean to be able to call its `evictSequenceAllocationCache()`.